app/assets/javascripts/components/locales/ja.jsx

```diff
@@ -39,8 +39,8 @@ const ja = {
   "tabs_bar.compose": "投稿",
   "tabs_bar.home": "ホーム",
   "tabs_bar.mentions": "返信",
-  "tabs_bar.local_timeline": "ローカルTL",
-  "tabs_bar.federated_timeline": "連合TL",
+  "tabs_bar.local_timeline": "ローカルタイムライン",
+  "tabs_bar.federated_timeline": "連合タイムライン",
```

Comment:
L42-L43: it will break at `600px < width < 650px`
![width](https://cloud.githubusercontent.com/assets/1874549/24979254/770faf80-200e-11e7-9974-cf0dbce7f389.gif)